src/components/reusables/ActionBar/ActionBarContent.module.css

```
@@ -0,0 +1,36 @@
+@import "@digdir/design-system-tokens/dist/digdir/tokens.css";
+
+.actionBarContent {
+  border-width: 1px;
+  border-style: solid;
+  border-bottom-left-radius: 4px;
+  border-bottom-right-radius: 4px;
+  border-top-left-radius: 0px;
+  border-top-right-radius: 0px;
+  background-color: #FFFFFF;
+}
+
+.actionBarContent.light {
+  border-color: var(--fds-semantic-border-neutral-subtle);
+  border-top-color: #FFFFFF;
```

If someone in the future make the inside a different color than white, this border will show.
I suggest changing the border-top to none instead. However, this is a nitpick so not stricktly a deal-breaker
```suggestion
  border-top: none;
```

---

Or possibly move this to the .actionBarContent instead so you don't have to repeat it for all the colors

---

yeah I tried border-top: none first but it didn't work. But I'll push new code that targets some of what you say here

---

Strange... It works fine for me.. 🤔